examples/python/blueprint/main.py

```diff
@@ -0,0 +1,47 @@
+"""Example of using the blueprint APIs to configure Rerun."""
+
+import argparse
+
+import numpy as np
+import rerun as rr
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Different options for how we might use blueprint")
+
+    parser.add_argument("--blueprint-only", action="store_true", help="Only send the blueprint")
+    parser.add_argument("--skip-blueprint", action="store_true", help="Don't send the blueprint")
+    parser.add_argument(
+        "--append-default", action="store_true", help="Append to the default blueprint instead of replacing it"
+    )
+    parser.add_argument("--auto-space-views", action="store_true", help="Automatically add space views")
+
+    args = parser.parse_args()
+
+    if args.blueprint_only:
+        # If only using blueprint, it's important to specify init_logging=False
+        rr.init("Blueprint", init_logging=False, add_to_app_default_blueprint=args.append_default, spawn=True)
+    else:
+        rr.init("Blueprint", add_to_app_default_blueprint=args.append_default, spawn=True)
```

Comment:
Not starting the blueprint recording at all when `--skip-blueprint` is set makes experimenting with the demo much more user-friendly:
```suggestion
        rr.init("Blueprint demo", init_blueprint=not args.skip_blueprint, add_to_app_default_blueprint=args.append_default, spawn=True)

Reply:
Can you elaborate?

init_blueprint=True is the default, so part of what I want to show here is the default behavior of initializing a blueprint but then not using it.